css-align-3/resources/alignment-parsing-utils.js

```diff
@@ -0,0 +1,19 @@
+var position = [ "start", "end", "self-start", "self-end", "left", "right", "center", "flex-start", "flex-end"];
+var distribution = [ "stretch", "space-around", "space-between", "space-evenly"];
+var baseline = [ "baseline", "last-baseline"];
+
+function checkPlaceContent(alignValue, justifyValue)
+{
+    var div = document.createElement("div");
+    div.setAttribute("style", "place-content: " + alignValue + " " + justifyValue)
```

Comment:
Not that when you call `checkPlaceContent()` with just 1 argument, this is expanded to: `"place-content: start undefined"`. Probably we don't want that.

Nit: Missing `;` at the end of the line.